app/uk/gov/hmrc/eoricommoncomponent/frontend/models/Service.scala

```diff
@@ -24,31 +24,42 @@ import uk.gov.hmrc.eoricommoncomponent.frontend.util.Constants
 case class Service(
   code: String,
   enrolmentKey: String,
+  shortName: String,
   callBack: String,
   friendlyName: String,
   friendlyNameWelsh: String
 )
 
 object Service {
 
-  val cds   = Service("cds", "HMRC-CUS-ORG", "", "", "")
-  val empty = Service("", "", "", "", "")
+  val cds = Service("cds", "HMRC-CUS-ORG", "", "", "", "")
+  // TODO Get rid of empty Service, throw exception in places where it's used
+  val empty = Service("", "", "", "", "", "")
 
-  val configuration = Configuration(ConfigFactory.load())
+  private val configuration = Configuration(ConfigFactory.load())
 
-  val supportedServices: Set[Service] =
+  private val supportedServices: Seq[Service] =
     configuration.get[Seq[Configuration]]("services-config").map { conf =>
       Service(
         code = conf.get[String]("name"),
         enrolmentKey = conf.get[String]("enrolment"),
+        shortName = conf.get[String]("shortName"),
         callBack = conf.get[String]("callBack"),
         friendlyName = conf.get[String]("friendlyName"),
         friendlyNameWelsh = conf.get[String]("friendlyNameWelsh")
       )
-    }.toSet
+    }
 
-  def withName(str: String): Option[Service] =
-    supportedServices.find(_.code == str)
+  val supportedServicesMap: Map[String, Service] = {
```

Comment:
Can this val be private?

Reply:
Yeah, of course. That's a good point. I thought that we're using this somewhere else.

Done